person.rb

```diff
@@ -1,18 +1,25 @@
-class Person
-  attr_accessor :name, :age
+require './nameable'
+
+class Person < Nameable
+  attr_accessor :age
   attr_reader :id
 
   def initialize(age, parent_permission: true, name: 'Unknown')
+    super()
```

Comment:
- [ ] Please see the `ArgumentError` in the Person class constructor due to your setting `name: 'Unknown'`. The implication of this is that you can no longer instantiate the Person class while supplying the `name` attribute like this: `Person.new(22, 'shedrackstormfield')`
![image](https://user-images.githubusercontent.com/46839548/231449456-5d631f15-1728-4afd-acf1-ed367b397d92.png)

NB: the name variable must not have a default value. Use the line below to get your program running correctly 😎 
```ruby
def initialize(age, name, parent_permission: true)
```